__test__/response.spec.ts

```diff
@@ -197,20 +197,70 @@ describe('Response', () => {
     }).toThrow();
   });
 
-  it('should fail if header is invalid', () => {
+  it('should fail if header value is invalid', () => {
     const responseMeta: ResponseMeta = {
       status: 200,
       header: {
         'content-type': 'application/json',
         version: [1, 2],
       },
     };
-    expect(() => {
+
+    let thrownError: ResponseValidationError;
+
+    // test validateResponseByPath
+    try {
       chowchow.validateResponseByPath('/header', 'get', responseMeta);
-    }).toThrow();
-    expect(() => {
+      fail('An exception should have been thrown');
+    } catch (error) {
+      thrownError = error;
+    }
+    expect(thrownError.toJSON().suggestions[0].error).toMatch(
```

Comment:
I did apply `toThrowErrorMatchingSnapshot` as you suggested. 

The one downside of this is that the test isn't verifying the nested "real" error that is "version: type should be string". Now, we're testing that the error is the more general "ResponseValidationError: Schema validation error".  Perhaps there is some other way to do this since toThrowErrorMatchingSnapshot is new to me.